Mathlib/LinearAlgebra/TensorProduct/Basic.lean

```diff
@@ -647,18 +152,6 @@ end UMP
 
 variable {M N}
 
-section
-
-variable (R M)
-
-/-- The base ring is a left identity for the tensor product of modules, up to linear equivalence.
--/
-protected def lid : R ⊗[R] M ≃ₗ[R] M :=
-  LinearEquiv.ofLinear (lift <| LinearMap.lsmul R M) (mk R R M 1) (LinearMap.ext fun _ => by simp)
-    (ext' fun r m => by simp; rw [← tmul_smul, ← smul_tmul, smul_eq_mul, mul_one])
-
-end
-
 @[simp]
 theorem lid_tmul (m : M) (r : R) : (TensorProduct.lid R M : R ⊗ M → M) (r ⊗ₜ m) = r • m :=
   rfl
```

Comment:
Do we really want to separate definitions from their `rfl` lemmas like this?

Reply:
(my inclination would be to leave the monoidal equivalences like `lid` / `assoc` / `comm` in `Basic.lean`, rather than separating them from their lemmas)